### ISSUE TITLE
Add solutions to CSES problemset

### ISSUE THREAD

**Issue by aksh1407** [1gne0u5/Learning-Resources#25]
Submit your solution after reading the [Submission Guidelines](https://github.com/1gne0u5/Learning-Resources/blob/master/Competitive%20Programming/readme.md).

Solutions are welcome in any language like C, C++, JAVA or Python but are *preferred in C++* considering its vast use cases in Competitive Coding.

**Comment by srajantiwari** [comment]
Hello, I would like to provide clean solutions with comments for better readability. Can I be assigned to this?

**Comment by TanmayKhule** [comment]
I can add _solution/CSEC/weirdAlgorithm_ and more introductory problems with clean code in **python**. Assign me the task

**Comment by aksh1407** [comment]
> I can add _solution/CSEC/weirdAlgorithm_ and more introductory problems with clean code in **python**. Assign me the task

Yeah sure! We would appreciate a few comments inside your code for better understanding.

**Comment by MsMore** [comment]
I would like to add  _**competitive Programming/CSEC/Sorting and Searching**_  in **C++** with clean code, proper comments and explaining the time complexity wherever necessary. Assign me.

**Comment by aksh1407** [comment]
> I would like to add _**competitive Programming/CSEC/Sorting and Searching**_ in **C++** with clean code, proper comments and explaining the time complexity wherever necessary. Assign me.

Your are good to go! Make sure to read the [Submission Guidelines](https://github.com/1gne0u5/Learning-Resources/tree/master/Competitive%20Programming#submissiom-guidelines).

**Comment by aksh1407** [comment]
The directory should be [CSES](https://github.com/1gne0u5/Learning-Resources/tree/master/Competitive%20Programming/Solutions/CSES) with separate folders created as per the categories of the CSES [problem set](https://cses.fi/problemset/). Solutions must be stored in their respective category's folder.

@TanmayKhule 
@MsMore

**Comment by MsMore** [comment]
@aksh1407  Review Pull Request #55 for the CSEC problem solutions.

**Comment by MsMore** [comment]
@aksh1407  Review Pull Request #64 for further CSEC problem solutions.

**Comment by TanmayKhule** [comment]
@aksh1407 I have closed #63 , review PR #67. Have made suggested changes.

**Comment by srajantiwari** [comment]
@aksh1407 Please review #85 , I've added a solution to the shortest routes 1 problem.
